page-sobre-nosotros.php

```diff
@@ -38,22 +38,34 @@
 
         <div class="margin-bottom-small" id="prensa">
           <h3 class="margin-bottom-small"><?php echo __('[:es]Prensa[:en]Press'); ?></h3>
-<?php
-      $current_query = current_query();
-      if ($current_query->have_posts()) {
-        while ($current_query->have_posts()) {
-          $current_query->the_post();
-/*           $number = get_post_meta($post->ID, '_igv_number'); */
-          $program_files = get_post_meta($post->ID, '_igv_program_files');
-
-          if (!empty($program_files[0])) {
-            echo '<ul>';
-            foreach ($program_files[0] as $file) {
-              echo "<li><a href=" . $file['file'] . " target='_blank' class='font-underline'>File: " . __($file['text']) . "</a></li>";
-            }
-            echo '</ul>';
+      <?php
+      $prensa_query = new WP_query( array(
+        'post_type' => 'programacion',
+        'posts_per_page' => -1,
+        'meta_key' => '_igv_number',
+        'orderby' => 'meta_value_num',
+        'order' => 'ASC',
+      ));
+
+      if ($prensa_query->have_posts()) {
+      ?>
+          <ul class="prensa-files">
+      <?php
+        while ($prensa_query->have_posts()) {
+          $prensa_query->the_post();
+          $number = get_post_meta($post->ID, '_igv_number');
+          $program_file = get_post_meta($post->ID, '_igv_program_file');
+          $color = get_post_meta($post->ID, '_igv_color');
+
+          if (!empty($program_file)) {
+          ?>
+            <li><a href="<?php echo $program_file[0]; ?>" target="_blank" rel="noopener" class="font-underline" style="color: <?php echo $color[0]; ?>">No. <?php echo $number[0]; ?></a></li>
```

Comment:
put `true` as the last arg in `get_post_meta()` for single values. then you dont need to reference the `[0]` index when echoing the value. `$color = get_post_meta($post->ID, '_igv_color', true);` like that